### DISCUSSION TITLE
Download events using node red

### DISCUSSION THREAD

**Discussion by gazzaman22** [blakeblackshear/frigate#8185] [Ask A Question] [answered]
Hi all,
 Ive just installed Frigate and have various cameras running and recording events, my question is how do I download a clip using Node red, I have never used MQTT  so this is all new to me
TIA

**Comment by NickM-27** [comment]
That can be done using this or other APIs https://docs.frigate.video/integrations/api#get-apieventsidclipmp4

**Reply by gazzaman22** [reply, marked as answer]
Thanks Ive been ready the documentation for the last few days and I dont understand it

**Reply by gazzaman22** [reply]
So I have sorted it out and can now email video clips by using node red, but its not every elegant as im having to use a delay  once the detection  count goes to 0  is there anyway of knowing when the recording as finished via the api 
TIA

**Reply by NickM-27** [reply]
that has been requested a couple times, it is possible in the future that an mqtt topic will be used to signal when an export has finished

**Reply by gazzaman22** [reply]
Thanks
